openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners_assessment.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n %}
+{% spaceless %}
+
+    {% block body %}
+    <div class="staff__grade__form ui-toggle-visibility__content" data-submission-uuid="{{ submission.uuid }}">
+        <div class="wrapper--staff-assessment">
+            <div>
+                <p>{% trans "Give this learner a grade using the problem's rubric." %}</p>
+            </div>
+
+            <div>
+                <article class="staff-assessment">
+                    <div class="staff-assessment__display">
+                        <header class="staff-assessment__display__header">
+                            <h4 class="staff-assessment__display__title">
+                                {% blocktrans %}
+                                    Response for: {{ student_username }}
+                                {% endblocktrans %}
+                            </h4>
+                        </header>
+
+                        {% include "openassessmentblock/oa_submission_answer.html" with answer=submission.answer answer_text_label="The learner's response to the question above:" %}
+
+                        {% include "openassessmentblock/oa_uploaded_file.html" with file_upload_type=file_upload_type file_url=staff_file_url header="Associated File" class_prefix="staff-assessment" show_warning="true" %}
+                    </div>
+
+                    <form class="staff-assessment__assessment" method="post">
+                        {% include "openassessmentblock/oa_rubric.html" with rubric_type="staff" rubric_feedback_prompt="(Optional) What aspects of this response stood out to you? What did it do well? How could it improve?" rubric_feedback_default_text="I noticed that this response..." %}
```

Comment:
TNL-3882 hasn't been done yet. I actually had never noticed that the text shows up here in the templates-- staff can set the text in Studio, and I assumed that text would be shown everywhere. It's very odd that the same text is used everywhere in the rubrics, and it might be different from what the students see. I'll talk to @andy-armstrong about how this ended up being hard-coded in the staff override form (which is where I took this HTML from...).